### PR TITLE
Create epitranscriptome_data.md

### PR DESCRIPTION
**Pull request by lauportell**

### CODE REVIEW

File: pages/your_domain/epitranscriptome_data.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+---
+title: Epitranscriptome data
+search_exclude: true
```
Comment (bedroesb) [inline pages/your_domain/epitranscriptome_data.md:3]:
```suggestion
```

Comment (bedroesb) [reply pages/your_domain/epitranscriptome_data.md:3]: otherwhise the page will not end up in the search box :)

**Review by bedroesb — APPROVED** [review]

File: _data/tags.yml [hunk 2]
```diff
@@ -37,6 +37,8 @@ struct info:
   url: structural_bioinformatics.html
 micro biotech:
   url: microbial_biotechnology.html
+epitrans data:
+  url: epitranscriptome_data.html
```
Comment (bedroesb) [inline _data/tags.yml:41]:
```suggestion
  url: epitranscriptome_data.html
```
```suggestion
epitrans:
  url: epitranscriptome_data.html
```

File: pages/your_domain/epitranscriptome_data.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+### Considerations
+Which kind of RNA modifications are you studying?
+
+### Solutions
+Upload your data and results in available databases. For not transient RNA modifications, REDIportal could be a solution.
+
+## Relevant tools and resources  
+<!--- Automatically generated table; edit the TAG below to the tag for this page, so that tools that have this page's tag are listed here. You can get the tag for this page from the [list of tags](https://github.com/elixir-europe/rdmkit/blob/master/_data/tags.yml). If it isn't listed there, please raise an issue.--->
+
+{% include toollist.html tag="epitrans data" %}
```
Comment (bedroesb) [inline pages/your_domain/epitranscriptome_data.md:49]:
```suggestion
{% include toollist.html tag="epitrans" %}
```

**Comment by bedroesb** [comment]
@lauportell I changed the tag to epitrans if that is fine! This way it is shorter and still recognizable (as suggested by @pinarpink )

**Comment by lauportell** [comment]
> @lauportell I changed the tag to epitrans if that is fine! This way it is shorter and still recognizable (as suggested by @pinarpink )

okay perfect, sounds good!

**Pull request merged**